src/Maestro/Maestro.Data/BuildAssetRegistryContext.cs

```diff
@@ -211,6 +212,19 @@ protected override void OnModelCreating(ModelBuilder builder)
                     })
                 .OnDelete(DeleteBehavior.Restrict);
 
+            builder.Entity<GoalTime>()
+              .HasKey(
+              gt => new
+              {
+                  gt.DefinitionId,
+                  gt.ChannelId
+              });
```

Comment:
Please fix the formatting here to match the one above.

Reply:
I think I fixed this. Let me know if this is ok..